Samples/EmbeddedAuthWithSDKs/EmbeddedAuthUITests/Utilities/Scenario.swift

```diff
@@ -254,9 +256,17 @@ protocol ScenarioValidator {
                     password: String,
                     firstName: String,
                     lastName: String,
+                    phoneNumber: String?,
+                    enrollFactors: [FactorType],
                     completion: @escaping (Error?) -> Void)
     func deleteUser(username: String,
                     completion: @escaping (Error?) -> Void)
+    func activatePolicy(named: String,
```

Comment:
Maybe we can use enum here? 
As we have here:

https://github.com/okta/okta-idx-swift/pull/41/files#diff-2253a42953da0df95499b2d21f9b7d56eea34e98771f76b28eba144a64ab7296R17